packages/react-diagrams-core/src/entities/port/PortModel.ts

```diff
@@ -70,17 +70,6 @@ export class PortModel<G extends PortModelGenerics = PortModelGenerics> extends
 		};
 	}
 
-	setPosition(point: Point);
-	setPosition(x: number, y: number);
-	setPosition(x, y?) {
-		let old = this.position;
-		super.setPosition(x, y);
-		_.forEach(this.getLinks(), (link) => {
-			let point = link.getPointForPort(this);
-			point.setPosition(point.getX() + x - old.x, point.getY() + y - old.y);
```

Comment:
It looks like setting position for points in `PortModel.setPosition` do nothing because x and old.x always match.

Reply:
I've returned this and I can't imagine why this does work and doesn't work without